ui/src/app/pages/profile/data-user-code-of-conduct.tsx

```diff
@@ -1,15 +1,53 @@
 import {Component} from '@angular/core';
```

Comment:
This is wrapped in Angular, so I can either 1) condition what content is shown within the file on the feature flag, or 2) branch what route we go to based on the feature flag in a few different places. I started out working on 1), but it would be relatively easy to switch to 2) if you, esteemed reviewer, think it would be cleaner.

Reply:
I like this approach, unless we think there is some need to display the v2 DUA for some reason after launching the v3 DUA.